CHANGELOG

```diff
@@ -1,3 +1,8 @@
+Release 137 (2022-11-01)
+------------------------
+Add nl.basjes/yauaa_context/jsonschema/1-0-4 (#1216)
+Bump igluctl version to allow schema lists in dev registry (#1236)
```

Comment:
Should we also add ?
`Add dev.amp.snowplow/amp_session/jsonschema/1-0-0 (#1235)`

Reply:
Thanks @igneel64 -- I had changed it locally and then I forgot to push it!!  Fixed now.